### ISSUE TITLE
Windows are not tiled correctly when one of the branches dissappears

### ISSUE THREAD

**Issue by nicolae-stroncea** [pop-os/shell#164]
![tiling_bug](https://user-images.githubusercontent.com/39338488/79284970-20da8580-7e8a-11ea-9e53-d1ddde4f0647.gif)

In the 1st case, the windows correctly switch orientation. In the second case, they remain their original size, even though they should switch.

**Comment by octos4murai** [comment]
I'd like to take a stab at this. To clarify, is the issue that the tiles are extended vertically side-by-side (as shown below) as opposed to top-and-bottom?
![tiles](https://user-images.githubusercontent.com/36859934/112740186-b3572600-8f2f-11eb-9049-6de9296872d8.png)

**Comment by nicolae-stroncea** [comment]
@octos4murai that's correct. Pop-os should adjust correctly by using the most intuitive layout, which does not happen in this case.
